generated/nidaqmx/_base_interpreter.py

```diff
@@ -1204,7 +1204,8 @@ def load_task(self, session_name):
 
     @abc.abstractmethod
     def read_analog_f64(
-            self, task, num_samps_per_chan, timeout, fill_mode, read_array):
+            self, task, num_samps_per_chan, timeout, fill_mode,
```

Comment:
This signature is in line with the grpc signature of this function and hence this is being followed.